config/manifests/bases/aerospike-kubernetes-operator.clusterserviceversion.yaml

```diff
@@ -35,8 +35,8 @@ spec:
ufdSvOa5MzV3PWukCT4eTUHdQREPQyRsoBdsIbbOrUPD5HYrrBWnoOOrg38hMqd0+kxXAqt68C0m9auSlPTPFJrNIUNfsBjoG+vJguGI4JDzsYI+bnsfxWmW55TS/oiBsBLov46vDnx6vOr3ZX63h1Walv6QDpkPAEqpijxrxVsC2Ith+QGAnwCAmPAjADljrUMJHitOs6Td9XmlduXXVO4cbbmayB2D8LA5Y1Yf1HPOAW2lcNaFNm4dfKyjyJVB6bzaijeKUy2fQGQqAOgmbTWAc8O9R0R/QcG0fGADliqF35tU0OWi1IwqUeqEgr0oz3qp0N2n4wsTEyOUwpPOUX4RaXFn+XdTbE7/ChSehzh2BpSu/i3PWvlf77ncNMuLwz2vqb5/zqmpuTAe2+hNfhdYui5/Cw2AUkog33PuaofVVPyiK9XyDYgkQ2RFScqMFbm1lw6NqRKRhQAW3vVphWYAow4sHSqvOM1y+zhPRr9SG0UkHgCgVL2m9f/cheKmOB9oIqM5q1gBYKrjoZo80otzqqteKTWnRypNvqUUpopIGASZAmRCZKMoU1txquWTElH/29WB/NsVJmckOB+bQvqmCEyZSsnXIJI+0Cd21Wd/ayxlj4aCzBPBCogMHrmIYG3ZtGlx2VeuNN1b6fLXwz8duA0AA8uXlKZZ1ukQ59miFruyFRWnpCQDQL/JpEPhLQADp7W1vwUwtsBSqhEKd5+vJLg8lmJFZDWA1XcoD1Boh6hqQH3Lpbk4gpuTZEVPGPH1N026uZnSNXI10GGt/NdjU6a8ZgsMe1gJViqFxYAkiiABIlECRCmFV4vMlpR8a8XPRtsvpmBVd/O3QE05+wSOflFQ38v/5FLFWPp8NESQAAigVDOAZojMgMjMvoDQXWXTpq3NvnKle8yF62rYMNI1tHp6+yYCvwosXcn/cH6RFRCoSdBBiOOkdK8CABU0+GLRHi0ypz+Qb638cLT1iMLPc60VPxzt+0akVCMEjQMbEASR1CHP/TaspuK7rs7wVjqKRMM6R79omwCMeNhSPGXmdIjMcf6uKX2vq00fOOR8aeAHpdNnLLZrpnzR8Cwg8yGiCdSPiqfM3JZ3tWp0gT5kj2agL7oBNCmFcxD1m7Ecfo+VUuqKSenP2u2mDglQhRAJh8jy3oDQ3yng6bEe+uZZK2aN1zZMZH4TWEXTMpZAsMb5u4hEAJh966tum9Yo+AkAV04n30IHgkb7HpcovCQ96qcAoAdrESJqB5zjcSLzFo3iTJJJZLcO/GBgO9OKzJa/z7dW/N9hqw/UfndznEy1hNVWnXGlrpKU9Dm5tZW3nBTIuXzpBIATAH5WlGppFpEYiASqQNN9wGj3QNU3h/4mSip1e39VvuMM6jhSdei3b8wZqLfEnPG0EvzF0cfyudJUSz1qKr5xb3X4N78JLAlQPxkMJF29heGO9zX5luNN2tqyaTPTs69UVY6qLudekLtpaMu9Xtkw8FtDSUr6U0rDGYiYRLCxJNXye9RUPONKUTk1F0+XpFpeUSKfd2yq/KAoNb1ft2n/7/bLbwYu2/nvEDw4uI26es6VvbmSVMt/U8DfFadafmq3yZ8/VXZ8ZqQOfXASpWhq1Jf+5FVX/LNH+nu0FE4ODclc68UdxWmW7wHyjwCgRL5ekmqpzq2p+KW3m2pUfhFYpakZ9+uClQAApdRI1wwOzN/ZDAC9Ju1nAB4eVYWini42pw9//Z9I5Wi/uEq/9fPKra38uDjF8gRMeH2g0KdLUi3HXS23v7PlK1p4zHwRuc/xdu3HWjAeK06z7BQlA+NCaoYObIYMuQJAV7/Mq618ZcR+d5yRfX7g119qwXimJNVyAJBLjo9CD7eL+pwMzEoHVGtgX/c47xW5Vejtf5FXXfGDolRLqoh8CQCUyPPFKRnn8movHhhNwcXm9BE/U9Wv/2o8xuq8yS8CS1fqeQxec4ER/zfW0Pd3OoI2A4BosqU0NeN+V2Z1DykhCxqyRmhTBYB7/p/WMdUg40fOy1+UyPOl5plnc6xVI17KsaqhoaM4MGoTgrR/gshTACCCJYAsUYNHxzcPk5VS/QJ8J89a8VNX2mbXpRKC3zhnnItgiYIsGXyBaBhyiNmv6Xj6ngamJ6i8moovl6RaZjgnscKk3ihOszyYV11xzOVCNG3EMUZTgNoDx1lcn+XzE0dLU1MzoTnPDCqbvVd+MtJ7cmpqLkCpwQFlHfrIF6pCRhX+Aqlz8ZWD5WoaTHd6RV7Nxe8AanBKgC6mt0qnZ05xpfS8q1WXA/u7vyS62gql9qo7nN1UCtXQ1S9F1/PzalwLKwDIt1aU59dcfE6UvhJK/yGgztyh7BtQ6mXR9XxX7o7g6DvP3GVkaP/era+He8/dvgMC2EN6Ox9XyjmJVqKh8Jqrn9Eo9HiiXyYSn9/DCrHZLndqWoZmMtlNpn573rVql8ZIgtvUE70RSAAAk8k+4mC2ZpcnlOghI5fsIEpz6ct1a7m25ru9Lqyn4wvdgRHfBgBdt4cnhWguj6MN7NX85Whc+gE9DJH2AEmww55pUmhXSrtgl94uWK03hrvX03ByayqLFFBcOjn9FwhC5NDnAnXYEkLUDUtllc3V8kx22xGlQmeOpS3Dstt+Lgh+eaS+vtt7hvtMs65ebSw0m5cH6EFhjtfqoUnBatjPSOxqVNto0m0+f/EzEREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREZHx/X/7J9jNSPekbAAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAyMC0wNy0xNFQwNjowMToyNyswMDowMN2Q520AAAAldEVYdGRhdGU6bW9kaWZ5ADIwMjAtMDctMTRUMDY6MDE6MjcrMDA6MDCszV/RAAAAAElFTkSuQmCC
```

Comment:
The original is an optimized webp scalable graphics image which renders well all scales. Unless this png is a different logo we can keep the webp image

Reply:
`webp` format is not supported. Atleast not during openshift certification.

Reply:
Can you try this. svg+xml is supported I think.

  - base64data: PHN2ZyB3aWR0aD0iMjUwMCIgaGVpZ2h0PSIyNTAwIiB2aWV3Qm94PSIwIDAgMjU2IDI1NiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiBwcmVzZXJ2ZUFzcGVjdFJhdGlvPSJ4TWlkWU1pZCI+PHBhdGggZD0iTTAgMGgyNTZ2MjU2SDBWMHoiIGZpbGw9IiNCMDI1MkEiLz48cGF0aCBkPSJNMTU2LjI5MyA5NS42MDVsLTczLjE2OSAzMi41OTQgNzMuMTcgMzIuODFWOTUuNjA1em0tOTIuMDMyIDM5Ljk3OWwtMTcuMDQ5LTcuMyAxNy4wNDktNy44ODIgMTQ0LjUyNy02NS4zNzZ2MTcuNDAzbC0zNy45MzIgMTYuODI0djc4LjExN2wzNy45MzIgMTd2MTYuNjA0TDY0LjI2IDEzNS41ODR6IiBmaWxsPSIjRkZGIi8+PC9zdmc+
    mediatype: image/svg+xml

Reply:
`svg+xml` works.